sdk/typescript/src/signers/txn-data-serializers/txn-data-serializer.ts

```diff
@@ -132,6 +162,20 @@ export type UnserializedSignableTransaction =
       data: PublishTransaction;
     };
 
+export type SpecialMoveCallTransaction =
```

Comment:
I really don't think we should have this either, I don't understand why we can't just have builders for standard move calls vs special casing all of this. It makes it much harder for this all to be portable.

Reply:
Thanks for the feedback! I've moved the functions to the `sui-system-state` utility class